packages/driver/cypress/integration/commands/actions/select_spec.js

```diff
@@ -365,7 +384,7 @@ describe('src/cy/commands/actions/select', () => {
           done()
         })
 
-        cy.get('#select-maps').select('de_dust2').select('de_aztec')
+        cy.get('#select-maps').select('de_aztec').select('de_dust2')
```

Comment:
2 tests that use `#select-maps` have been changed because it tries to select the already-selected first option. I changed them to select the second one.